archive_viewer/mixins/plot_config.py

```diff
@@ -1,6 +1,8 @@
 from widgets import ColorButton
 from pyqtgraph import ViewBox
-
+from typing import Dict
+from qtpy.QtGui import QColor
+from time import sleep
```

Comment:
Style: Reorder imports and remove the unused import. I typically follow this order:

1. full library imports (`import re`)
2. standard library imports (`from os import path`)
3. qtpy imports (`from qtpy.QtCore import Qt`)
4. pyqtgrah imports
5. pydm imports (`from pydm.widgets import PyDMPushButton`)
6. local module imports (`from config import logger`)

When I have multiple imports that match a category, I sort them by library name length. e.g.:
```
from qtpy.QtGui import QColor, QBrush, QPen
from qtpy.QtWidgets import QPushButton
```

All that to say I'd order these imports like this:
```
from typing import Dict
from qtpy.QtGui import QColor
from pyqtgraph import ViewBox
from widgets import ColorButton
```